async_upnp_client/aiohttp.py

```diff
@@ -26,6 +27,22 @@
 _LOGGER_TRAFFIC_UPNP = logging.getLogger("async_upnp_client.traffic.upnp")
 
 
+def _fixed_host_header(url: str) -> Dict[str, str]:
+    """Strip scope_id from IPv6 host, if needed."""
+    url_parts = urlparse(url)
```

Comment:
We will probably see a lot host headers, so its likely a lot cheaper to see if "%" if in the url before parsing it and reject right unless we expect most urls to contain a `%`